recipes/r-recetox-aplcms/meta.yaml

```
@@ -22,36 +22,48 @@ build:
   
 requirements:
   host:
-    - r-base
+    - r-base >=3.5
```

```suggestion
    - r-base
```
Redundant, since we only build for the two latest R versions anyway.